### PR TITLE
feat: iframe integration doc

### PR DESCRIPTION
**Pull request by eswarclynn**

### CODE REVIEW

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+That's it. You are all done.
+
+## Iframe Integration
+
+If you want a quick integration of a fully featured video conferencing app inside your application within a day, iframe is the way to go.
+
+As discussed above, you could fork and deploy our [sample web app](https://github.com/100mslive/100ms-web/) and use that URL in an iframe or use one of 100ms' links from the [rooms page](https://dashboard.100ms.live/rooms) in your dashboard to make it even quicker.
+
+```html
+<iframe
```
Comment (Nixon-Samuel) [inline docs/javascript/v2/guides/deployment-iframe-integration.mdx:89]: It'd be helpful if we give an example for src. 
Also, an example of customizing the UI and allowed values for "allow" would help.

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+title: Deployment and Iframe Integration
+nav: 2.7
+---
+
```
Comment (Nixon-Samuel) [inline docs/javascript/v2/guides/deployment-iframe-integration.mdx:5]: adding an overview section to inform the users of the end result with the outline of steps to be followed will be good.

**Review by arjunattam — APPROVED** [review]
content looks fine to start. let's align this with the #1130 with a following structure

* get started with sample app (intro page in #1130)
   * deployment steps for x (content from this PR)
   * example customisations (content in #1130)
   * embed it with iframe (content from this PR)

given we have nested headings, will this be doable? @Nixon-Samuel can we merge this into #1130 and ship the above structure?

**Comment by Nixon-Samuel** [comment]
> 

sure @arjunattam

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+## Deploying 100ms Sample Web App
+
+### Building the app
+
+[100ms-web](https://github.com/100mslive/100ms-web/) app is just like any react application. To build the app, just run
+
+```bash
+yarn build
```
Comment (saikatmitra91) [inline docs/javascript/v2/guides/deployment-iframe-integration.mdx:13]: npm alternate ?

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx (hunk 2 above)
Comment (saikatmitra91) [reply docs/javascript/v2/guides/deployment-iframe-integration.mdx:5]: and mentioning the benefits of using iframe is something folks will love :)

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+-   [Netlify Tutorial - Drag and drop deploys on Netlify
+    (Video)](https://youtu.be/etZ9HSUoTPU)
+
+> We recommend using Git in general if you are planning to use this code for production. Only use the direct method if you want to quickly test something or do some proof-of-concept.
+
+If you are interested in Netlify's CI/CD or just want to learn more about deploying to Netlify, please check their docs [here](https://docs.netlify.com/site-deploys/create-deploys/)
+
+### Vercel
+
+Deploying to Vercel is pretty much straightforward similar to Netlify. You can start with a [New Vercel app](https://vercel.com/new).
```
Comment (saikatmitra91) [inline docs/javascript/v2/guides/deployment-iframe-integration.mdx:57]: we should talk about configuring environment variables too

File: docs/javascript/v2/guides/deployment-iframe-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+### Deploying the code
+
+There are many ways to deploy 100ms-web. We discuss a few methods below. Please feel free to reach out to us if you don't find your preferred platform here.
+
+-   [Netlify](#netlify)
+-   [Vercel](#vercel)
+
+### Netlify
+
+Deploying 100ms-web to [Netlify](https://www.netlify.com/) is straightforward and same as deploying any project to Netlify. There are two ways to deploy to Netlify
```
Comment (saikatmitra91) [inline docs/javascript/v2/guides/deployment-iframe-integration.mdx:34]: we should talk about how to configure environment variable too

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

**Comment by saikatmitra91** [comment]
Screenshot 2023-01-30 at 10 07 39 AM
Vercel and Netlify should be inside the deploying the code

**Comment by Nixon-Samuel** [comment]
> 

This is the latest [preview link](https://100ms-docs-bnt1xzyv3-100mslive.vercel.app/docs/javascript/v2/get-started/react-sample-app/build-and-deploy#deploy-the-code) for this.

**Comment by eswarclynn** [comment]
will be fixed by #1130 (build-and-deploy, embed-with-iframe)

**Pull request closed without merging**